includes/pages/user_settings.php

```diff
@@ -184,7 +184,7 @@ function user_settings() {
                   form_text('mail', _("E-Mail") . ' ' . entry_required(), $mail),
                   form_checkbox('email_shiftinfo', _("Please send me an email if my shifts change"), $email_shiftinfo),
                   form_text('jabber', _("Jabber"), $jabber),
-                  form_text('hometown', _("Hometown"), $hometown),
+                  form_text('kommentar', _("additional Info(Language / Profession)"), $kommentar),
```

Comment:
at least (mind the space): **A**dditional Info (Language / Profession)
Although I'd prefer to spell out Information